src/components/multiselect.vue

```diff
@@ -20,7 +20,7 @@ except according to the terms contained in the LICENSE file.
       @keydown="toggleAfterEnter" @mousedown.prevent>
       <option value="">{{ selectOption }}</option>
     </select>
-    <span class="form-label">{{ label }}</span>
+    <span class="form-label" aria-hidden="true">{{ label }}</span>
```

Comment:
The `aria-label` of the `<select>` element is set to `label`, so `label` doesn't need to be surfaced to screen readers here.